### ISSUE TITLE
[emerg] upsync_server: upsync_type invalid para when upsync_type=consul_services

### ISSUE THREAD

**Issue by msmmbl** [xiaokai-wang/nginx-stream-upsync-module#31]
Here is my nginx config file

```
stream {
  upstream shadowsocks {
    upsync 127.0.0.1:8500/v1/catalog/service/shadowsocks upsync_timeout=6m upsync_interval=25000ms upsync_type=consul_services strong_dependency=off;
    upsync_dump_path /etc/nginx/stream_conf.d/servers_consul.conf;

    include /etc/nginx/stream_conf.d/servers_consul.conf;
  }

  server {
    listen 41000;
    listen 41000 udp;

    proxy_pass shadowsocks;
  }

  server {
    listen 41001;
    upstream_show;
  }
}
```

but nginx fail to start
```
7月 18 14:56:32 debian systemd[1]: Starting A high performance web server and a reverse proxy server...
7月 18 14:56:33 debian nginx[594]: nginx: [emerg] upsync_server: upsync_type invalid para in /etc/nginx/stream_conf.d/shadowsocks.conf:3
7月 18 14:56:33 debian nginx[594]: nginx: configuration file /etc/nginx/nginx.conf test failed
7月 18 14:56:33 debian systemd[1]: nginx.service: Control process exited, code=exited status=1
7月 18 14:56:33 debian systemd[1]: Failed to start A high performance web server and a reverse proxy server.
7月 18 14:56:33 debian systemd[1]: nginx.service: Unit entered failed state.
7月 18 14:56:33 debian systemd[1]: nginx.service: Failed with result 'exit-code'.
```

I try use `upsync_type=consul`, it works
but `upsync_type=consul_services`, it failed

**Comment by gfrankliu** [comment]
You have to use the latest master code. Don't use v1.1.0.
The consul_services was added after v1.1.0 was release.
@xiaokai-wang : shall we cut a new release?

**Comment by xiaokai-wang** [comment]
@gfrankliu ok, that will be good.

**Comment by gfrankliu** [comment]
Released v1.2.0
